README.md

```diff
@@ -203,7 +203,7 @@ Other Resources
 -------------
 - Core SDK Nuget Package: https://www.nuget.org/packages/Box.V2/
 - JWT Support Nuget Package: https://www.nuget.org/packages/Box.V2.JWTAuth/
-- Box Windows SDK Video Tutorial: https://developers.box.com/box-windows-sdk-tutorial/
```

Comment:
This link redirects to docs.box.com but the tutorial is not hosted on that site. Change this link to point to the YouTube version to resolve the broken link.